WcaOnRails/app/models/competition.rb

```diff
@@ -519,6 +525,10 @@ def has_fees?
     base_entry_fee_lowest_denomination + competition_events.sum(:fee_lowest_denomination) > 0
   end
 
+  def competitor_limit_enabled?
```

Comment:
nit: I think this can be accomplished with a call to `alias_method`, and rubocop might even complain at you if you don't.